tab_donate.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Donate
+displaytext: Donation Packages
+layout: null
+tab: true
+order: 2
+tags: mstg
+---
+
+# Donations
+
+If you'd like to support the Mobile Security Testing project we'll be happy to accept your donations. You may donate 5, 500, 5000 (UDS/EUR/GBP) or any custom amount. Certain amounts allow you to opt-in for a "[Donation Package](#donation-packages)" (see below).
+
+100% of the funds raised go directly to the OWASP foundation and allow us to ask for budget to fund our project activities:
+
+- Editing and proofreading by professional editors.
+- Graphic design and layout improvements.
+- Improving our automated delivery pipelines.
+- Acquiring new test devices if certain tests can not be executed on existing emulators/simulators/test devices.
+
+**You may do an online donation directly from the OWASP website by clicking the DONATE button above.**
+
+When you click on the DONATE button and fill in the form be sure to verify that it has the line:
+"Publicly list me as a supporter of OWASP Mobile Security Testing Guide"
+
+<img style="padding: 10px;" width="500px" src="assets/images/donation_form.png" />
+
+> Please note that the OWASP Donation Policy has changed since 22-Sept-2020. All details can be found in [OWASP Donations Policy](https://owasp.org/www-policy/operational/donations) page.
+
+## Donation Packages
+
+If you'd like to opt-in for a Donation Package please contact [Sven Schleier](mailto:sven.schleier@owasp.org) or [Carlos Holguera](mailto:carlos.holguera@owasp.org) to verify your donation. We will then, together with the OWASP Foundation verify and process it.
+
+> These types of public recognition shall be online no less than one year, or in the case of Project no less than the next major release, whichever is greater.
+
+### Good Samaritan (USD 500)
+
+- Listed as supporter on the project website and GitHub
+- Listed as supporter in the printed and ebook versions.
+- 5 Paperback Books.
```

Comment:
I think we should reduce the amount of paperback books drastically. One book costs 35 USD (https://www.lulu.com/en/en/shop/bernhard-müller-and-jeroen-willemsen-and-sven-schleier/owasp-mobile-security-testing-guide/paperback/product-1q2rerwp.html?page=1&pageSize=4). Otherwise we are supporting Lulu.com more than OWASP :-) 

- Good Samaritan: 1
- Honorourable Benefactor: 3
- God Mode Sponsor: 5

What do you think Carlos?